emission/integrationTests/suggestionsys/category_of_business_nominatim.dataset.json

```diff
@@ -9,5 +9,126 @@
             }
         },
         "output": ["deptstores", "menscloth", "womenscloth"]
+    },
+    
+    {
+        "test_name": "Manhattan Beach Kettle",
+        "test_type": "Non-chain restaurant in suburban downtown area",
+        "input": {
+            "loc": {
+                "coordinates": [-118.4097008, 33.8853681],
+                "type": "Point"
+            }
+        },
+        "output": ["diners", "tradamerican"]
+    },
+
+    {
+        "test_name": "Manhattan Beach Beckers",
+        "test_type": "Suburban bakery, non-chain",
+        "input": {
+            "loc": {
+                "coordinates": [-118.4106739, 33.8842780],
+                "type": "Point"
+            }
+        },
+        "output": ["bakeries", "sandwiches", "coffee"]
+    },
+
+    {
+        "test_name": "Manhattan Beach FISHBAR", 
+        "test_type": "Suburban local bar/restaurant, non-chain",
+        "input": {
+            "loc": {
+                "coordinates": [-118.4182952, 33.9017119], 
+                "type": "Point"
+            }
+        },
+        "output": ["seafood", "breakfast_brunch", "newamerican"]
+    },
+
+    {
+        "test_name": "El Segundo Umi",
+        "test_type": "Sushi restaurant in small shopping center",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3947103, 33.9024449],
+                "type": "Point"
+            }
+        },
+        "output": ["japanese", "sushi", "seafood"]
+    },
+
+    {
+        "test_name": "Redondo Beach Captain Kidd's",
+        "test_type": "Fish market in seaside town",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3923475, 33.8434248],
+                "type": "Point"
+            }
+        },
+        "output": ["seafoodmarkets", "seafood", "fishnchips"]
+    },
+
+    {
+        "test_name": "Mira Costa High School Manhattan Beach",
+        "test_type": "Local public school in Los Angeles County",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3893806, 33.8729666],
+                "type": "Point"
+            }
+        },
+        "output": []
```

Comment:
Are you sure about this?
https://www.yelp.com/biz/mira-costa-high-school-manhattan-beach

```
In [2]: sugg.business_reviews(sugg.YELP_API_KEY, "mira-costa-high-school-manhattan-beach
   ...: ")
{'alias': 'mira-costa-high-school-manhattan-beach',
 'categories': [{'alias': 'highschools',
   'title': 'Middle Schools & High Schools'}],
```